### PR TITLE
Added the localization middleware

### PR DESCRIPTION
**Pull request by Scarpy019**
The localization middleware is run before all controllers and it sets a `locale` interface to `res.locals.lang`. Language is selected from a `lang` cookie that defaults to `lv` in case the cookie is empty or invalid as a locale.

### CODE REVIEW

**Review by EPoikans — APPROVED** [review]
Looks good for frontend, awaiting the look over for any conflicts with cookies & tokens.

File: source/middleware/LocalizationMiddleware.ts [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+import type locale from '../localization/localization.js';
+
+const localizations: Record<string, locale> = {
+	en,
+	lv
+};
+
+export function localization (req: Request, res: Response, next: NextFunction): void {
+	const lang = req.cookies.lang;
+	console.log(localizations);
```
Comment (towergame) [inline source/middleware/LocalizationMiddleware.ts:13]: Did you mean to leave this `console.log` in?

**Review by towergame — CHANGES_REQUESTED** [review]
Looks good to me, aside from that one `console.log` that seems to be left there :P

File: source/middleware/LocalizationMiddleware.ts (hunk 1 above)
Comment (Scarpy019) [reply source/middleware/LocalizationMiddleware.ts:13]: No, i did not
I'll remove it

**Review by towergame — APPROVED** [review]
:+1:

**Pull request merged**